CHANGELOG.md

```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.1.2] Preview Release
+### Added
+- Added support for JDK 11
+- Added support for Jacoco in VSTS
```

Comment:
Maybe change to "Updated Maven dependencies, library versions, and Jacoco for Java 11"? That would also move it to the "Changed" section.